tick/stateful/functions.go

```diff
@@ -573,6 +790,24 @@ func (integer) Call(args ...interface{}) (v interface{}, err error) {
 	return
 }
 
+var integerFuncSignature = map[Domain]ast.ValueType{}
+
+func initializeIntegerFuncSignature() {
+	d := Domain{}
+	d[0] = ast.TBool
+	integerFuncSignature[d] = ast.TInt
+	d[0] = ast.TString
+	integerFuncSignature[d] = ast.TInt
+	d[0] = ast.TInt
+	integerFuncSignature[d] = ast.TInt
+	d[0] = ast.TFloat
+	integerFuncSignature[d] = ast.TInt
```

Comment:
Reusing the same domain in this function works right? I would expect so but wanted to confirm.